client/my-sites/promote-post-i2/components/post-item/style.scss

```diff
@@ -113,6 +113,9 @@ body.is-section-promote-post-i2 {
 
 		&__post-promote-button {
 			border-radius: 4px;
+			border: 1.5px solid var(--color-accent) !important;
```

Comment:
I think the `! important` here is not needed. It is always better to try and avoid using `important` if possible.
Same for the one below, I tested it in the browser and it appears to be working fine

![Screenshot 2024-01-24 at 16 39 43](https://github.com/Automattic/wp-calypso/assets/1258162/a994e822-ddd7-4071-94f7-c53259de948b)

Reply:
I'll remove it :)